### PR TITLE
Update dbstructure

### PR DESCRIPTION
**Pull request by aidan-hubley**
TL;DR

In meeting we discussed need for counting number of times a post has been flagged / reported. I figured this meant we need a db item. Check it out.

Separately, we want users to be able to hide individual posts and users from their feed. We are picturing a local storage solution, with a list of hidden user ids and /or post ids.

@dev-arekusandoru thoughts?

### CODE REVIEW

File: dbstructure.json
Comment (dev-arekusandoru) [inline dbstructure.json]:
```suggestion
                    "flag_count": 0
```
i think avoiding abbreviations where possible is a good practice

Comment (02NRA) [reply dbstructure.json]: @aidan-hubley I agree with Alex on the abbreviation front; I'll approve this PR once his change is merged (I didn't know this was a feature in Github so we never made a policy for it, but I'll assume we're doing the same as we would for merging PRs, meaning in this case, Aidan needs to make the change)

**Review by 02NRA — APPROVED** [review]
Thank you for taking my commented advice :)

**Pull request merged**
